Mixpanel/Mixpanel.m

```diff
@@ -254,9 +254,10 @@ + (void)assertPropertyTypesInArray:(NSArray *)arrayOfProperties depth:(NSUIntege
 
 - (NSString *)defaultDistinctId
 {    
-    NSString *distinctId = [self IFA];
+    NSString *distinctId;
 
 #if !defined(MIXPANEL_WATCHOS) && !defined(MIXPANEL_MACOS)
+    distinctId = [self IFA];
```

Comment:
If you build for Mac, you would assign a value to distinctId and never use it, which causes a static analyzer warning.